packages/config/src/options.ts

```diff
@@ -223,6 +221,12 @@ const driverConfigOptions: Array<DriverConfigOption> = [
     validation: validate.isBoolean,
     isExperimental: true,
     requireRestartOnChange: 'server',
+  }, {
+    name: 'experimentalStudio',
+    defaultValue: false,
+    validation: validate.isBoolean,
+    isExperimental: true,
+    requireRestartOnChange: 'server',
```

Comment:
What is changing on the server side that requires a server restart?

Reply:
Nothing actually - good point. It looks like we do need to restart the browser though to re-initialize the `Reporter` with `config.experimentalStudio`'s new value, so I set this to `'browser'`.